--- car_gui_menu_patch/gamemodes/laird.pwn	2026-06-16 13:42:59.815259285 +0000
+++ navigator_gui_patch/gamemodes/laird.pwn	2026-06-16 14:01:19.462961794 +0000
@@ -3510,6 +3510,20 @@
 	DIALOG_CAR_GUI_INSURANCE,
 	DIALOG_CAR_GUI_REPAIR_CONFIRM,
 	DIALOG_CAR_GUI_UNLOAD_CONFIRM,
+	DIALOG_NAV_GUI_MAIN,
+	DIALOG_NAV_GUI_IMPORTANT,
+	DIALOG_NAV_GUI_NEWBIE_JOBS,
+	DIALOG_NAV_GUI_JOBS,
+	DIALOG_NAV_GUI_STATE_ORGS,
+	DIALOG_NAV_GUI_CRIME_ORGS,
+	DIALOG_NAV_GUI_TRANSPORT,
+	DIALOG_NAV_GUI_ENTERTAINMENT,
+	DIALOG_NAV_GUI_BANKS,
+	DIALOG_NAV_GUI_PIERS,
+	DIALOG_NAV_GUI_AUTOSALONS,
+	DIALOG_NAV_GUI_CITIES,
+	DIALOG_NAV_GUI_STATIONS,
+	DIALOG_NAV_GUI_NEAREST,
 	DIALOG_FAMILY_CAR_SELL,
 	DIALOG_OWNABLE_CAR_NUMBER,
 	DIALOG_OWNABLE_CAR_PT,
@@ -7623,6 +7637,7 @@
 #include "../include/system/blackpass.inc"
 #include "../include/system/used_vehicle_market_ext.inc"
 #include "../include/system/car_gui_menu.inc"
+#include "../include/system/navigator_gui.inc"
 #include "../gamemodes/gamegui/death_gui.inc"
 #include "../gamemodes/sile/ipacket.inc"
@@ -14148,6 +14163,7 @@
     if(BlackMarket_HandleDialog(playerid, dialogid, response, listitem, inputtext)) return 1;
     if(Trade_HandleDialog(playerid, dialogid, response, listitem, inputtext)) return 1;
     if(CarGUI_HandleDialog(playerid, dialogid, response, listitem, inputtext)) return 1;
+    if(NavGUI_HandleDialog(playerid, dialogid, response, listitem, inputtext)) return 1;
 	if(dialogid == DIALOG_ADM_STDI)
 	{
 		if(response)
@@ -47716,34 +47732,9 @@
 
 CMD:gps(playerid, params[])
 {
-	DisablePlayerGPS(playerid);
-
-	new sile_title[128];
-	format(sile_title, sizeof sile_title, "{FF5252}%s{ffffff} | GPS", NameServer[SILE_NAME]);
-
-	Dialog
-	(
-		playerid, DIALOG_GPS, DIALOG_STYLE_LIST,
-		sile_title,
-		""c_server"1. "c_b"Важные места\n"\
-		""c_server"2. "c_b"Работы для новичков\n"\
-		""c_server"3. "c_b"Основные работы\n"\
-		""c_server"4. "c_b"Государственные организации\n"\
-		""c_server"5. "c_b"Преступные группировки\n"\
-		""c_server"6. "c_b"Таксопарки\n"\
-		""c_server"7. "c_b"Развлечения\n"\
-		""c_server"8. "c_b"Банки\n"\
-		""c_server"9. "c_b"Причалы\n"\
-		""c_server"10. "c_b"Автосалоны\n"\
-		""c_server"11. "c_b"СТО и тюнинг-ателье\n"\
-		""c_server"12. "c_b"Города и населенные пункты\n"\
-		""c_server"13. "c_b"Вокзалы\n"\
-		""c_server"14. "c_b"Поиск ближайших мест",
-		"Выбрать", "Закрыть"
-	);
-	return 1;
+    #pragma unused params
+    return NavGUI_ShowMain(playerid);
 }
-
 CMD:help(playerid, params[])
 {
 	Dialog
