--- work_wheel/gamemodes/laird.pwn	2026-06-16 14:01:19.000000000 +0000
+++ fortune_wheel_gui_patch/gamemodes/laird.pwn	2026-06-16 19:06:36.381104659 +0000
@@ -7638,6 +7638,7 @@
 #include "../include/system/used_vehicle_market_ext.inc"
 #include "../include/system/car_gui_menu.inc"
 #include "../include/system/navigator_gui.inc"
+#include "../include/system/fortune_wheel_gui.inc"
 #include "../gamemodes/gamegui/death_gui.inc"
 #include "../gamemodes/sile/ipacket.inc"
@@ -8199,6 +8200,7 @@
     g_admin_auth_dialog_open[playerid] = false;
     report_OnPlayerDisconnect(playerid);
     Inventory_Save(playerid);
+    Fortune_Cleanup(playerid);
     DeletePlayerTextDraw(playerid);
     Kosti_OnPlayerDisconnect(playerid);
     if(IsPlayerLogged(playerid)) SavePlayerBP(playerid);
@@ -33996,6 +33998,9 @@
 if(playertextid == PlayerText:INVALID_TEXT_DRAW)
         return 0;
 
+if(Fortune_OnPlayerClickPlayerTextDraw(playerid, playertextid))
+        return 1;
+
 if(Kosti_OnPlayerClickPlayerTextDraw(playerid, playertextid))
         return 1;
 
